data/radar/2024-06-13/Prisma.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Prisma"
+ring: trial
```

Comment:
should we move to `hold`?  I don't think we want to do any new projects with Prisma, right?